MODULE.bazel

```diff
@@ -0,0 +1,22 @@
+module(
+    name = "com_google_googleapis",
+    version = "0.0.1",
+)
+
+bazel_dep(name = "protobuf", version = "21.7", repo_name = "com_google_protobuf")
+bazel_dep(name = "rules_proto", version = "5.3.0-21.7")
+
+switched_rules = use_extension("//:extensions.bzl", "switched_rules")
+switched_rules.use_languages(
+    cc = True,
```

Comment:
I would start small here since the way this is implemented every of these language rulesets will need to be in the BCR and a dep of this module to work with strict deps. We could start with `cc`, `java`, `python` and `go` for now.